python/ray/autoscaler/cli_logger.py

```diff
@@ -513,6 +513,30 @@ def confirm(self, yes, msg, *args, _abort=False, _default=False, **kwargs):
 
         return res
 
+    def prompt(self, msg: str, *args, **kwargs):
+        """Display a confirmation dialog.
+
+        Args:
+            msg (str): The mesage to display to the user before the prompt.
+        """
+        if self.old_style:
+            return
+
+        complete_str = cf.underlined(msg + ":") + " "
+
+        self._print(complete_str, linefeed=False)
```

Comment:
I see we need `linefeed=False` here so we couldn't just use `print`. The downside is that we have to call `_format_msg(msg, *args, **kwargs)` ourselves since `_print` does not format the message it prints.